mods/ts/mod.yaml

```diff
@@ -270,12 +271,21 @@ ModContent:
 			Download: basefiles
 		music: Base Game Music
 			TestFiles: ^Content/ts/scores.mix
-			Sources: tibsun, tibsun-linux, tfd, origin
+			Sources: tibsun, tibsun-linux, tfd, origin, firestorm
 			Download: music
+		expansion: Firestorm Expansion
+			TestFiles: ^Content/ts/expand01.mix
+			Sources: firestorm, tfd
+			Required: true
+			Download: expansion
+		fs-music: Firestorm Expansion Music
```

Comment:
https://itunes.apple.com/album/command-conquer-tiberian-sun/id83190565 is still being sold so I am a bit hesitant.